src/main/sensors/gyro.c

```diff
@@ -44,20 +45,29 @@ float gyroADCf[XYZ_AXIS_COUNT];
 static int32_t gyroZero[XYZ_AXIS_COUNT] = { 0, 0, 0 };
 static const gyroConfig_t *gyroConfig;
 static biquadFilter_t gyroFilter[XYZ_AXIS_COUNT];
+static biquadFilter_t gyroFilterNotch[XYZ_AXIS_COUNT];
+static biquadFilter_t gyroFilterOld[XYZ_AXIS_COUNT];
```

Comment:
You are right, I added gyroFilterOld for debugging purposes in order to directly compare the old and the new filter. It won't be necessary for normal flight.